source/pervasive/ptr.rs

```diff
@@ -362,7 +365,8 @@ impl<V> PPtr<V> {
         opens_invariants_none();
 
         unsafe {
-            dealloc(self.uptr as *mut u8, Layout::for_value(&*self.uptr));
+            #[cfg(not(feature = "no_global_allocator"))]
```

Comment:
This function won't make any sense if this line is removed. Can you just disable the entire `ptr` module for `no_global_allocator`?

Reply:
That makes sense.